types/src/v18/wallet/error.rs

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: CC0-1.0
+
+use core::fmt;
+
+use bitcoin::amount::ParseAmountError;
+use bitcoin::{address, hex};
+
+use crate::error::write_err;
+use crate::NumericError;
+
+/// Error when converting a `ListUnspentItem` type into the model type.
+#[derive(Debug)]
+pub enum ListUnspentItemError {
```

Comment:
We can just re-use the v17 error because its the same (account is not converted so does not contribute to the error).